step.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -ex
 
-curl -Ls https://install.tuist.io | bash
+if ! [ -x "$(command -v tuist)" ]; then
```

Comment:
Just checking if this will work for projects that bundle tuist binary locally

Reply:
You mean if they don't have it in their PATH? my guess is it won't, but it wouldn't be a regression, it would just install it the same way as it did before.